src/app/asset-page/asset-page.component.ts

```diff
@@ -866,40 +853,56 @@ export class AssetPage implements OnInit, OnDestroy {
         return false
     }
 
-    /**
-     * trackItemView
+  /**
+   * trackEvent
+   * send captain's log events associated with asset
+   * @param asset
+   * @param eventType
+   * @returns void
+   */
+  private trackEvent(asset: Asset, eventType: string): void {
+    const hasAccess = !this.showAccessDeniedModal
+    const reasonForAuth = this.showAccessDeniedModal ? "not_authorized" : "license"
```

Comment:
I was able to simplify the `reasonForAuth` check. After discussing with auth team, all content on Artstor is managed via a license. Therefore even public/open images are technically managed by a license, and we can say the `reasonForAuth` is `license`.